### DISCUSSION TITLE
Gesture to go back ios.

### DISCUSSION THREAD

**Discussion by farhazulMullick** [Tlaster/PreCompose#246] [General]
@Tlaster Can we implement a gesture behaviour to go back on iOS?

**Comment by Tlaster** [comment]
There's a `swipeProperties` parameter in `NavHost`, which enables the swipe back gesture, but currently have some knowing issue #197

**Reply by farhazulMullick** [reply]
Can you present any examples ? @Tlaster

**Reply by Tlaster** [reply]
Just set the `swipeProperties` and you're good to go
